src/Symfony/Bundle/TwigBundle/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 CHANGELOG
 =========
 
-4.4.0
+4.0.0
 -----
 
  * removed `ContainerAwareRuntimeLoader`
+ * added priority to twig extensions 
```

Comment:
should be added in a new entry for 4.1.0

Reply:
@nicolas-grekas done :)